build.gradle

```diff
@@ -19,6 +19,8 @@ dependencies {
 	implementation 'org.springframework.security:spring-security-crypto'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation 'org.webjars:bootstrap:4.6.0-1'
+	implementation 'org.webjars:webjars-locator-core:0.46'
```

Comment:
I'm not familiar with this JAR. What does it do? Is it a dependency of the bootstrap JAR?